### ISSUE TITLE
fix: Remove second tab stop for <HyperLink> wrapper around Card component

### ISSUE THREAD

**Issue by TheEarlyNerd** [openedx/paragon#2200]
Currently the Card component has an `isClickable` prop that correctly makes the tab stop select the whole card. However, there is a secondary implementation where you wrap a component that does not have the `isClickable` prop with a link and it should operate similarly to the version of the card with the `isClickable` prop.

## Acceptance Criteria
- ensure the second implementation of the Card wrapped in a link has only one tab stop with the correct selection styles.

**Comment by adamstankiewicz** [comment]
Closed via https://github.com/openedx/paragon/pull/2290
